client/my-sites/upgrades/domain-management/edit/mapped-domain.jsx

```diff
@@ -117,4 +117,5 @@ const MappedDomain = React.createClass( {
 	}
 } );
 
+export { MappedDomain };
```

Comment:
This can just be `export MappedDomain`; or `export const MappedDomain = //...` above